llvm/lib/Transforms/Coroutines/CoroFrame.cpp

```diff
@@ -2896,6 +2960,43 @@ void coro::salvageDebugInfo(
   }
 }
 
+void coro::salvageDebugInfo(
+    SmallDenseMap<Argument *, AllocaInst *, 4> &ArgToAllocaMap, DPValue *DPV,
```

Comment:
`DPV` is assumed to be non-null, let's express that at the API level by making it a reference